src/lib/es5.d.ts

```diff
@@ -536,6 +536,11 @@ interface BooleanConstructor {
     readonly prototype: Boolean;
 }
 
+interface BooleanConverter {

```

Comment:
The reason for adding `BooleanConverter` is to be specific as possible, rather than forcing another function to choose which member of `BooleanConstructor` is the right one.